aspnetcore/aspnetcore-2.1.md

```diff
@@ -0,0 +1,169 @@
+---
+title: What's new in ASP.NET Core 2.1
+author: isaac2004
+description: Learn about the new features in ASP.NET Core 2.1.
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: riande
+ms.custom: mvc
+ms.date: 5/29/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: aspnetcore-2.1
+---
+# What's new in ASP.NET Core 2.1
+
+This article highlights the most significant changes in ASP.NET Core 2.1, with links to relevant documentation.
+
+## SignalR
+
+SignalR has been rewritten for ASP.NET Core 2.1. ASP.NET Core SignalR includes a number of improvements:
+
+* A simplified scale-out model.
+* A new JavaScript client with no jQuery dependency.
+* A new compact binary protocol based on MessagePack.
+* Support for custom protocols.
+* A new streaming response model.
+* Support for clients based on bare WebSockets.
+
+For more information, see [ASP.NET Core SignalR](xref:signalr/index)
+
+## Razor class libraries
+
+ASP.NET Core 2.1 makes it easier to build and include Razor-based UI in a library and share it across multiple projects. The new Razor SDK enables building Razor files into a class library project that can be packaged into a NuGet package. Views and pages in libraries are automatically discovered and can be overridden by the application. By integrating Razor compilation into the build:
+
+* The app startup time is significantly faster.
+* Fast updates to Razor views and pages at runtime are still available as part of an iterative development workflow.
+
+For more information, see [Create reusable UI using the Razor Class Library project](xref:mvc/razor-pages/ui-class)
+
+## Identity UI library & scaffolding
+
+ASP.NET Core 2.1 provides [ASP.NET Core Identity](xref:security/authentication/identity) as a [Razor Class Library](xref:mvc/razor-pages/ui-class). Applications that include Identity can apply the scaffolder to selectively add the source code contained in the Identity Razor Class Library (RCL). You might want to generate source code so you can modify the code and change the behavior. For example, you could instruct the scaffolder to generate the code used in registration. Generated code takes precedence over the same code in the Identity RCL.
```

Comment:
Applications that include Identity can apply the scaffolder -> Applications that include Identity can apply the new Identity scaffolder